modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -129,6 +129,8 @@ An installer-provisioned installation requires two static virtual IP (VIP) addre
 
 You specify these IP addresses when you install the {product-title} cluster.
 
+When you create your own DNS or DHCP, you must, in addition to a VIP address for the API and ingress, set the fully qualified domain name (FQDN) of each node, that is of the bootstrap, controller, and worker.
+
 [id="installation-nutanix-installer-infra-reqs-dns-records_{context}"]
 === DNS records
 You must create DNS records for two static IP addresses in the appropriate DNS server for the Nutanix instance that hosts your {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster.
```

Comment:
I would add the new content after this line, before the table. I would phrase it like this:

"If you use your own DNS/DHCP server, you must also create records for each node, including the bootstrap, control plane, and compute nodes."

Reply:
@brendan-daly-red-hat Did you want to update anything here based on @bscott-rh 's review before merge?